cephprimarystorage/cephprimarystorage/cephdriver.py

```
@@ -40,24 +40,23 @@ def create_volume(self, cmd, rsp, agent=None):
             return rsp
 
-        if ceph.is_xsky():
-            # do NOT round to MB
+        multiple_of_MB = cmd.size % sizeunit.m == 0
+        if cmd.format == 'qcow2':
+            vol_path = "rbd:%s" % path
+            linux.qcow2_create(vol_path, cmd.size, False)
+        elif multiple_of_MB:
+            call_string = 'rbd create --size %s --image-format 2 %s' % (sizeunit.Byte.toMegaByte(cmd.size), path)
+            call_string = self._wrap_shareable_cmd(cmd, call_string)
+            shell.call(call_string)
```

Ruff (0.8.2)

`shell` may be undefined, or defined from star imports

(F405)